stratum/templates/podsecuritypolicy-stratum.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.podsecuritypolicy.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: 1-stratum
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
```

Comment:
Is this required?

Reply:
If the privileged is true the other one must be true as well, otherwise it doesn't work. (I tested this)